### ISSUE TITLE
Rebase event-exporter on scratch

### ISSUE THREAD

**Issue by tallclair** [GoogleCloudPlatform/k8s-stackdriver#207]
It looks like event-exporter only requires a base image to provide ca-certificates. If that's the case, the image should be rebased on scratch, grabbing the ca-certs from a multi-stage dockerfile, like so:

(copied from [coredns](https://github.com/coredns/coredns/blob/master/Dockerfile))
```
FROM gcr.io/google-containers/debian-base-amd64:0.3.2
RUN clean-install ca-certificates

FROM scratch
COPY --from=0 /etc/ssl/certs /etc/ssl/certs
EXPOSE 80
COPY event-exporter /
```

**Comment by x13n** [comment]
I think this is true for more projects in this repository than just event-exporter.

**Comment by tallclair** [comment]
Yes, definitely. I filed separate issues for all the ones I found.
